### ISSUE TITLE
Some question about the result

### ISSUE THREAD

**Issue by Zora137** [noahzn/Lite-Mono#133]
hello,sorry to bother you many times
this is the result I run your code by this command without pretain

>  python train.py --data_path path/to/your/data --model_name mytrain --num_epochs 30 --batch_size 12   --lr 0.0001 5e-6 31 0.0001 1e-5 31

![image](https://github.com/noahzn/Lite-Mono/assets/113323676/be7cef4e-d459-4405-8e26-209a926cb71d)
 
It cannot reach the resulin your papert without pretain:
![image](https://github.com/noahzn/Lite-Mono/assets/113323676/69976277-a460-425e-a606-c66c08fc6fbb)

is this normal？ or did dosomething wrong?
expect your answer，thank you so much

**Comment by noahzn** [comment]
Hi, the results are too bad. As stated in our paper: 

> For models trained from scratch an initial learning rate of 5e−4 with a cosine learning rate schedule [26] is adopted, and the training epoch is set to 35. 

Could you please try using a larger initial learning rate?

**Comment by 757787182** [comment]
> Hi, the results are too bad. As stated in our paper:
> 
> > For models trained from scratch an initial learning rate of 5e−4 with a cosine learning rate schedule [26] is adopted, and the training epoch is set to 35.
> 
> Could you please try using a larger initial learning rate?

hi nice work !!!
can you show me your args file ,I do many times the ruseltalways this

![QQ图片20240418140654](https://github.com/noahzn/Lite-Mono/assets/83005849/a852d30f-b8bb-416a-be67-13cad8bc76dc)
 trained by lite-mono

**Comment by noahzn** [comment]
> > Hi, the results are too bad. As stated in our paper:
> > > For models trained from scratch an initial learning rate of 5e−4 with a cosine learning rate schedule [26] is adopted, and the training epoch is set to 35.
> > 
> > 
> > Could you please try using a larger initial learning rate?
> 
> hi nice work !!! can you show me your args file ,I do many times the ruseltalways this
> 
> ![QQ图片20240418140654](https://private-user-images.githubusercontent.com/83005849/323484745-a852d30f-b8bb-416a-be67-13cad8bc76dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM0MjM3NDQsIm5iZiI6MTcxMzQyMzQ0NCwicGF0aCI6Ii84MzAwNTg0OS8zMjM0ODQ3NDUtYTg1MmQzMGYtYjhiYi00MTZhLWJlNjctMTNjYWQ4YmM3NmRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE4VDA2NTcyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhMjBmZTVhZDFjN2VjODNmNTAzMTRmZDM0OTBiNDRlYzQyY2E1NDEzMjdlMDBiMjQzNTNlMjc1NzhmZGE1MTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WsUa3dXYZfRSbQHlxqx1KQjURV7bG6LrYlZvxHO97r8) trained by lite-mono

Hi, you can try setting the learning rate to `--lr 0.0001 5e-6 16 0.0001 1e-5 16`. `drop_path` can be set to `0.3`. But this might cause your training not converging. Please make sure you are using the same dependencies as we used. https://github.com/noahzn/Lite-Mono/issues/58

Also, please check the results of each epoch, not only the last epoch. The best result should be achieved at an earlier epoch.

**Comment by 757787182** [comment]
> > > 嗨，结果太糟糕了。正如我们的论文所述：
> > > > 对于从头开始训练的模型，采用余弦学习率计划[26]的初始学习率为5e−4，训练周期设置为35。
> > > 
> > > 
> > > 你能尝试使用更大的初始学习率吗？
> > 
> > 
> > 嗨，干得好!!你能给我看看你的args文件吗，我做了很多次ruselt总是这个
> > ![QQ图片20240418140654](https://private-user-images.githubusercontent.com/83005849/323484745-a852d30f-b8bb-416a-be67-13cad8bc76dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM0MjM3NDQsIm5iZiI6MTcxMzQyMzQ0NCwicGF0aCI6Ii84MzAwNTg0OS8zMjM0ODQ3NDUtYTg1MmQzMGYtYjhiYi00MTZhLWJlNjctMTNjYWQ4YmM3NmRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE4VDA2NTcyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhMjBmZTVhZDFjN2VjODNmNTAzMTRmZDM0OTBiNDRlYzQyY2E1NDEzMjdlMDBiMjQzNTNlMjc1NzhmZGE1MTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WsUa3dXYZfRSbQHlxqx1KQjURV7bG6LrYlZvxHO97r8)由 Lite-Mono 训练
> 
> 嗨，您可以尝试将学习率设置为。 可以设置为 。但这可能会导致您的训练无法收敛。请确保您使用的依赖项与我们使用的依赖项相同。#58`--lr 0.0001 5e-6 16 0.0001 1e-5 16``drop_path``0.3`
> 
> 另外，请检查每个纪元的结果，而不仅仅是最后一个纪元。最好的结果应该在更早的时代实现。
ok thanks!!! i will try it

**Comment by zaobaqiemanke** [comment]
> > Hi, the results are too bad. As stated in our paper:
> > > For models trained from scratch an initial learning rate of 5e−4 with a cosine learning rate schedule [26] is adopted, and the training epoch is set to 35.
> > 
> > 
> > Could you please try using a larger initial learning rate?
> 
> hi nice work !!! can you show me your args file ,I do many times the ruseltalways this
> 
> ![QQ图片20240418140654](https://github.com/noahzn/Lite-Mono/assets/83005849/a852d30f-b8bb-416a-be67-13cad8bc76dc) trained by lite-mono

I would like to ask if this result uses pre training weights, that is, if the -- mypretrain file is set as the pth file
